Batch File/Batch File.sublime-syntax

```diff
@@ -1983,7 +1995,14 @@ contexts:
   line-continuations:
     - match: \^\n
       scope: punctuation.separator.continuation.line.dosbatch
-      push: bol-pop
+      push: line-continuation-body
+
+  line-continuation-body:
+    # The first linefeed diretly following a continuation is treated literal.
+    # Note: Must consume \n to work.
+    - match: ^\n
```

Comment:
I believe this doesn't solve the "leading caret after a continuation" problem:

```batchfile
::: `@` is not escaped here, the second `^` is
:::^
^
@echo This shouldn't be marked as a comment.
```

```suggestion
    - match: ^\^
      pop: 1
    - match: ^\n
```

I'm not sure whether the caret should be scoped as an escape character (as it will be styled even within comments).